spec/features/residents_see_nice_errors_spec.rb

```diff
@@ -1,6 +1,39 @@
 require 'rails_helper'
 
 RSpec.feature 'Error pages' do
+  scenario 'when an application error occurs' do
+    runtime_error = RuntimeError.new("They're coming outta the walls! They're coming outta the goddamn walls!")
+    allow(StartForm).to receive(:new)
+      .and_raise runtime_error
+
+    allow(Rails.logger).to receive(:error)
+    allow(Rollbar).to receive(:error)
+
+    visit '/'
+    expect { click_on 'Start' }.to raise_error runtime_error # ...and rely on rails default routing to render the error
```

Comment:
This test looks to assert when visiting the root path the StartForm is called and raises. Would it be more useful to assert that that we see content from our new 500 error page?

Reply:
We can't: Capybara shows errors in test mode.

Agreed this test case isn't great. What I'm trying to make sure is that generic errors don't get somehow silently rescued, and instead bubble all the way to the top. Maybe this test doesn't actually give us that?

I think it's possible to temporarily disable error rendering, but I think that ended up seeming like a massive faff